pkg/kubelet/config/flags.go

```diff
@@ -81,7 +81,7 @@ type ContainerRuntimeOptions struct {
 
 func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	// General settings.
-	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt'.")
+	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt', 'remote'.")
```

Comment:
Could you also add a deprecation warning for 'rkt'?
s/rkt/rkt (deprecated)

Reply:
Sure. 
For posterity this is related to #58418.